ontology/EBUCorePlus/ebucoreplus.owl

```diff
@@ -1986,6 +1993,11 @@ ec:hasIssuer rdf:type owl:ObjectProperty ;
                         "Émetteur"@fr .
 
 
+###  http://www.ebu.ch/metadata/ontologies/ebucoreplus#hasJob
+ec:hasJob rdf:type owl:ObjectProperty ;
+          owl:inverseOf ec:isOrderedBy .
+
+
```

Comment:
Should it be : 
ec:hasJob (an agent has a job)
ec:isPerformedBy (a job is performed by an agent)

Reply:
The original property was "isOrderedBy", in the use: ProductionJob isOrderedBy Contract.
We wanted to add an inverse property to that, because often a ProductionJob needs to refer to a Contract, eg. to access a contract ID or an order ID. This could be expressed as "Contract hasJob ProductionJob", therefore the "hasJob" property. 
Please note: There is no Agent involved, because Agents and  ProductionJob are connected only through Involvement, not directly.
I think the requested change can be executed as proposed.